adb_graphics/datahandler/grib.py

```diff
@@ -328,26 +328,35 @@ def values(self, level=None, name=None, **kwargs) -> np.ndarray:
         elif len(field.shape) == 3:
 
             # Available variable levels
-            try:
-                levs = self.contents.variables[field.dimensions[0]][::]
-
-                # Requested level
-                lev_val, lev_unit = self.numeric_level()
-                lev_val = lev_val * 100. if lev_unit == 'mb' else lev_val
-
-                # The index of the reqested level
+            dim_name = spec.get('vertical_level_name',
+                                field.dimensions[0])
+            print('grib.py: dim_name = ', dim_name)
+            levs = self.contents.variables[dim_name][::]
+            print('grib.py: levs = ', levs)
+
```

Comment:
Are the print statements meant for debugging, or do we want to leave them here? I see some below, too.